### PR TITLE
Update logs page + user form

### PR DESCRIPTION
**Pull request by juliasaveliff**
- Fix margins on logs page
- Update user form
    - Add card
    - Show all sites (not just datacenters) -> new endpoint on backend
    -  Refactor some of the code

### CODE REVIEW

File: rackcity/views/site_views.py [hunk 1]
```diff
@@ -13,6 +13,15 @@
 from rest_framework.permissions import IsAuthenticated
 
 
+@api_view(["POST"])
+@permission_classes([IsAuthenticated])
+def site_all(request):
+    """
+    Return list of all sites (datacenters and offline storage sites).
+    """
+    return get_many_response(Site, SiteSerializer, "sites", request)
```
Comment (benauriemma) [inline rackcity/views/site_views.py:22]: What a handy method

File: src/components/userView/user.scss [hunk 2]
```diff
@@ -3,3 +3,11 @@
   margin-top: 0 !important;
   overflow: hidden;
 }
+
+.permission-card {
+  background-color: #202b33 !important;
```
Comment (benauriemma) [inline src/components/userView/user.scss:8]: important!

File: src/forms/userForm.tsx [hunk 3]
```diff
@@ -181,11 +176,11 @@ class UserForm extends React.Component<UserFormProps, UserFormState> {
                       show_asset_options: false,
                     });
                   }}
-                ></Checkbox>
+                />
```
Comment (benauriemma) [inline src/forms/userForm.tsx:179]: Nice.

Comment (brendancheng20) [reply src/forms/userForm.tsx:179]: man whoever wrote `<Checkbox ></Checkbox>` for ev3 must feel like a clown rn

File: src/forms/userForm.tsx [hunk 4]
```diff
@@ -326,60 +273,102 @@ class UserForm extends React.Component<UserFormProps, UserFormState> {
...
+            label={AssetPermissionSelection.PER_SITE}
+            value={AssetPermissionSelection.PER_SITE}
+          />
+        </RadioGroup>
+        {this.state.asset_permission_selection ===
+        AssetPermissionSelection.PER_SITE
+          ? this.renderSiteChecks()
+          : null}
+      </Card>
+    );
```
Comment (brendancheng20) [inline src/forms/userForm.tsx:327]: I like this refactor a lot, way cleaner than what I had initially

File: src/forms/userForm.tsx (hunk 3 above)
Comment (juliasaveliff) [reply src/forms/userForm.tsx:179]: HAHA I didn't even know what this but pycharm was really yelling at me for it

File: src/forms/userForm.tsx (hunk 4 above)
Comment (juliasaveliff) [reply src/forms/userForm.tsx:327]: cheers!

**Pull request merged**